src/net_processing.cpp

```diff
@@ -1252,6 +1253,39 @@ bool PeerManagerImpl::BlockRequestAllowed(const CBlockIndex* pindex)
            (GetBlockProofEquivalentTime(*pindexBestHeader, *pindex, *pindexBestHeader, m_chainparams.GetConsensus()) < STALE_RELAY_AGE_LIMIT);
 }
 
+bool PeerManagerImpl::FetchBlock(NodeId id, const CBlockIndex* index)
+{
```

Comment:
Consider adding some line breaks to this function to split up the units of logic. It's a bit of a wall of text right now.

Reply:
I added some line breaks and prose.